superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -922,12 +928,14 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
   const onDashboardChange = (dashboard: SelectValue) => {
     updateAlertState('dashboard', dashboard || undefined);
     updateAlertState('chart', null);
+    updateEmailSubject();
```

Comment:
Instead of updating the subject line here, you can also add the formatting when saving.

Reply:
Modified